Example-iOS/ViewController.swift

```diff
@@ -17,7 +17,25 @@ class ViewController: UIViewController {
         let path = Bundle.main.path(forResource: "test", ofType: "md")!
         let document = CMDocument(contentsOfFile: path, options: CMDocumentOptions(rawValue: 0))
         
-        let renderer = CMAttributedStringRenderer(document: document, attributes: CMTextAttributes())!
+        let textAttributes = CMTextAttributes()!;
```

Comment:
here too

Reply:
Also done in commit a5db7ab81f364a6605b601c1a8bf222fa9fb4322